wlauto/workloads/gmail/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -61,7 +62,7 @@ public void clickNewMail() throws Exception {
 
     public void setToField() throws Exception {
         Timer result = new Timer();
-        UiObject toField = getUiObjectByDescription("To", "android.widget.TextView");
+        UiObject toField = getUiObjectByText("To", "android.widget.TextView");
```

Comment:
This is the one that works everywhere.   On some devices the description is empty - yes, really!